modeltranslation/tests/tests.py

```diff
@@ -990,6 +992,38 @@ def test_reverse_relations(self):
         self.assertIn(fk_inst_en, test_inst2.test_fks.all())
         self.assertNotIn(fk_inst_de, test_inst2.test_fks.all())
 
+    def test_reverse_lookup_with_filtered_queryset_manager(self):
```

Comment:
This test should be failing with master branch, but it works. Are you sure you testing what you think you are testing?

Reply:
Guess not. Ill have another look